### PR TITLE
Added note for existing HTTP PROXY configurations

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
Added note for existing HTTP PROXY configurations
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [ ] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

### CODE REVIEW

**Comment by joel-hamill** [comment]
TODO: once approved, add to Enterprise

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 1]
```diff
@@ -183,12 +183,17 @@ This required parameter specifies a YAML nested list (`-`) of DNS resolvers for
 
 This parameters specifies whether to enable the DC/OS proxy.
 
-*  `use_proxy: 'false'` Do not configure DC/OS [components](/docs/1.9/overview/components/) to use a custom proxy. This is the default value.
-*  `use_proxy: 'true'` Configure DC/OS [components](/docs/1.9/overview/components/) to use a custom proxy. If you specify `use_proxy: 'true'`, you can also specify these parameters:
-    **Important:** The specified proxies must be resolvable from the provided list of [resolvers](#resolvers).
+*  `use_proxy: 'false'` Do not configure DC/OS [components](/docs/1.8/overview/components/) to use a custom proxy. This is the default value.
+*  `use_proxy: 'true'` Configure DC/OS [components](/docs/1.8/overview/components/) to use a custom proxy. If you specify `use_proxy: 'true'`, you can also specify these parameters:
```
Comment (cruhland) [inline 1.9/administration/installing/custom/configuration-parameters.md]: On this line and the one above, why did you change the links to 1.8 docs, even though this page is for 1.9?

**Review by cruhland — COMMENTED** [review]
LGTM, had one question.

File: 1.9/administration/installing/custom/configuration-parameters.md (hunk 1 above)
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: Good catch, and a mistake :)
